xls/public/runtime_build_actions.h

```diff
@@ -86,6 +86,20 @@ absl::StatusOr<std::string> ProtoToDslx(absl::string_view proto_def,
                                         absl::string_view text_proto,
                                         absl::string_view binding_name);
 
+
+// Converts the specified XLS IR text into a combinational Verilog module.
+//
+// Args:
+//  ir: Text for the XLS IR package.
+//  path: Path to use for source location information for the given text.
+//    Since text may be generated, an empty string or a pseudo path like
+//    "<generated>" is acceptable.
+//  entry: Entry function for the XLS IR package,
+//    if empty entry function of the package is used.
+//  module_name: Name of the Verilog module,
+//    if empty use the mangled IR function name.
+absl::StatusOr<std::string> ConvertIrToCombinationalVerilog(absl::string_view ir, absl::string_view path, absl::string_view entry, absl::string_view module_name);
```

Comment:
should we make combinational vs pipeline different entrypoint or a separate argument? (I opted for the former because the arguments needed a significantly differents)

Reply:
In think the end-goal should be that there is a single entry point and that whether or not it's combinational/procedural; the entry point; and any generation constraints (ex. stages, input/output flops, signal names, inverting/non-inverting reset, etc...) can be configured.

That said, once we get into generating IR for a set of cooperating procs the needs would become a bit more concrete.

Reply:
I like really the idea of defining the public interface payloads thru proto, should we go the extra-mile and include a [`service` definition](https://developers.google.com/protocol-buffers/docs/proto3#services)?

I think this could help with:
- tracking changes made to the API and versioning
- help with binding/parameters marshaling for various language (could pass paramater as raw byte and decode using code generated from proto)
- expose the toolchain as a service

WDYT?